index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE HTML>
+<html lang="en">
+  <head>
+    <title>Infrastructure Club Travel Guide</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css"
+     integrity="sha256-kLaT2GOSpHechhsozzB+flnD+zUyjE2LlfWPgU04xyI="
+     crossorigin=""/>
+    <script src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
+     integrity="sha256-WBkoXOwTeyKclOHuWtc+i2uENFpDZ9YPdf5Hf+D7ewM="
+     crossorigin=""></script>
+    
+    <style>
+      html, body, div { margin: 0; padding 0;}
+      #map { height: 100svh; }
+    </style>
+  </head>
+  
+  <body>
+    <div id="map"></div>
+    
+    <script>
+      var map = L.map('map').fitWorld();
+      L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+        maxZoom: 19,
+        attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+      }).addTo(map);
+      
+      fetch("./data/map.geojson")
+        .then(response => {
+          if (!response.ok) {
+           throw new Error("HTTP error " + response.status);
+          }
+          return response.json();
+        })
+        .then(geojson => {
+          L.geoJSON(geojson).bindPopup(function (layer) {
+            if (layer.feature.properties && layer.feature.properties.name) {
+              name = layer.feature.properties.name;
+            } else {
+              name = "Unnamed Point"
+            }
+            
+            if (layer.feature.properties && layer.feature.properties.description) {
+              description = layer.feature.properties.description;
+            } else {
+              description = ""
+            }
+            
+            // TODO: XSS in the JSON?
+            return "<h2>" + name + "</h2><p>" + description + "</p>"
```

Comment:
it might be worth doing something here before merging, even if it's minimal :)

I think you can return HTMLElement here, so you can do this:

```suggestion
            var title = document.createElement("h2");
            title.innerHTML = name;
            var paragraph = document.createElement("p");
            paragraph.innerHTML = description;
            var div = document.createElement("div");
            div.appendChild(title);
            div.appendChild(paragraph);
            return div;
```

Reply:
innerHTML has the same problem :)
I'll push a fix though